README.md

```diff
@@ -103,19 +103,6 @@ If you would like to build and try  Gluten with **ClickHouse** backend, please f
 
 [Gluten Usage](./docs/get-started/GlutenUsage.md) listed the parameters and their default value of build command for your reference
 
-### 3.2.4 Jar conflicts
```

Comment:
I am moving this part to velox-backend-troubleshooting.md. Suppose a CH troubleshooting doc can be separated maintained and including this part. cc @zzcclp.